### ISSUE TITLE
Server Client doesn't set Content-Type header

### ISSUE THREAD

**Issue by xtellurian** [microsoft/tsiclient#158]
I'm intercepting requests made by the server client to a TSI Environment so I can use the same environment in a multi-tenant way.

When the client makes a request to the server (i.e. to the 'environment'), it puts JSON in the body but doesn't set the "Content-Type" to application/json. This means intercepting and validating the JSON is hard, because in my case ASP .NET Core framework returns 415 unsupported media type.

Currently working around by reading the body, but this is not ideal.

Expected Result:
* Requests containing JSON should have the Content-Type = application/json

Actual Result
* Content-Type header is "text/plain;charset=UTF-8"

**Comment by darsney** [comment]
ahh yeah this is good feedback, thanks!  gimme a day or two to take a look, we are working on settling the 1.3.0 release right now so things are a bit swamped...appreciate the feedback!

**Comment by xtellurian** [comment]
For anyone having the same issue, I'm using a custom resource filter to fix the content type before it hits the controller.

```cs
public class AddJsonContentType : Attribute, IResourceFilter
{
        public void OnResourceExecuting(ResourceExecutingContext context)
        {
            context.HttpContext.Request.Headers["Content-Type"] = "application/json";

        }

        public void OnResourceExecuted(ResourceExecutedContext context)
        {
        }
}
```

```cs
[AddJsonContentType]
public async Task<IActionResult> QueryTimeSeries(QueryRequest query)
{
...
}

```

**Comment by darsney** [comment]
haven't forgot about you i promise, @niusoff  should be adding this in a PR this week

**Comment by darsney** [comment]
this has been merged to master, see [here](https://github.com/microsoft/tsiclient/commit/967888ef1886ce833b13843d9450da86868617cd#diff-ba93fd7d2ed1bdd990c97b27d8eec3b9R39)

It's been published in a [beta release, 1.3.6-beta.0](https://www.npmjs.com/package/tsiclient/v/1.3.6-beta.0)

This will be included in the next non-beta release...feel free to pull it down and give us feedback.  Thanks!
